contracts/FlowIDTableStaking.cdc

```diff
@@ -626,6 +657,51 @@ pub contract FlowIDTableStaking {
     /// and pay rewards to nodes at the end of an epoch
     pub resource Admin {
 
+        /// Any node can call this function to register a new Node
```

Comment:
Is this comment still accurate if this is in the `Admin` now?

Reply:
Nope, I'll change it